lib/perl/Genome/Qc/Tool/Picard.pm

```diff
@@ -20,9 +20,9 @@ sub get_metrics {
 
     my $file = $self->qc_metrics_file;
     my $gmt_class = $self->gmt_class;
-    my %metrics = $self->metrics;
     my $metric_results = $gmt_class->parse_file_into_metrics_hashref($file);
-    return $self->_get_metrics(\%metrics, $metric_results);
+    my $metric_header_as_key = $gmt_class->can('_metric_header_as_key') ? $gmt_class->_metric_header_as_key : undef;
+    return $self->_flatten_metrics_hash($metric_results, metric_header_as_key => $metric_header_as_key);
```

Comment:
I've added another commit to my PR that adds a mechanism for parsing multiple accumulations from the metrics to the metric parsing in `Genome::Model::Tools::Picard`.  It still needs to be wired up to everything else to be useful.